flatpak/org.xorg.adriconf.json

```diff
@@ -0,0 +1,183 @@
+                    "sha256": "6e74fcba0d245451c58fc8a196e9d103789bc510e1eee1a9b1e816c5209e79a9"
+                }
+            ]
+        },
+
+		{
+			"name": "pangomm",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://ftp.acc.umu.se/pub/GNOME/sources/pangomm/2.40/pangomm-2.40.1.tar.xz",
+					"sha256": "9762ee2a2d5781be6797448d4dd2383ce14907159b30bc12bf6b08e7227be3af"
+				}
+			]
+		},
+
+		{
+			"name": "atkmm",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://ftp.acc.umu.se/pub/GNOME/sources/atkmm/2.24/atkmm-2.24.2.tar.xz",
+					"sha256": "ff95385759e2af23828d4056356f25376cfabc41e690ac1df055371537e458bd"
+				}
+			]
+		},
+
+		{
+			"name": "graphene-gobject-1.0",
+			"buildsystem": "meson",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://archive.ubuntu.com/ubuntu/pool/main/g/graphene/graphene_1.8.0.orig.tar.gz",
+					"md5": "d0ee65f4bdd30a5db4fd8d2ac46cd8cc"
+				}
+			]
+		},
+
+		{
+			"name": "gtk+-4.0",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/gtk+4.0/3.90.0-0ubuntu2/gtk+4.0_3.90.0.orig.tar.xz",
+					"sha256": "7bb33b4101a4b3e51dc286000d61d1130575c3e7dee88656f4b43fc2a5a05b97"
+				}
+			]
+		},
+
+		{
+			"name": "gtkmm",
+			"buildsystem": "autotools",
+			"sources": [
+				{
+				"type": "archive",
+				"url": "https://ftp.acc.umu.se/pub/GNOME/sources/gtkmm/3.22/gtkmm-3.22.2.tar.xz",
+				"sha256": "91afd98a31519536f5f397c2d79696e3d53143b80b75778521ca7b48cb280090"
+				}
+			]
+		},
+
+		{
+			"name": "libxml++-2.6",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/libxml++2.6/2.40.1-2/libxml++2.6_2.40.1.orig.tar.xz",
+                    "sha256": "4ad4abdd3258874f61c2e2a41d08e9930677976d303653cd1670d3e9f35463e9"
+                }
+            ]
+		},
+
+		{
+			"name": "libpci",
+			"buildsystem": "simple",
+			"sources": [
+				{			
+					"type": "archive",
+					"url": "https://www.kernel.org/pub/software/utils/pciutils/pciutils-3.6.2.tar.xz",
+					"md5": "77963796d1be4f451b83e6da28ba4f82"
+				}
+			],
+			"build-commands": [
+				"make PREFIX=/app SHAREDIR=/app/share/hwdata SHARED=yes",
+				"make PREFIX=/app SHAREDIR=/app/share/hwdata SHARED=yes install install-lib",
+				"chmod -v 755 /app/lib/libpci.so"
+			]
+		},
+
+		{
+			"name": "adriconf",
+			"buildsystem": "cmake",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/jlHertel/adriconf",
+					"branch": "master"
+				}
+			]
+		}
+	],
+
+	"finish-args": [
+		/*In future we need wayland also!*/
+   		"--socket=x11",
+   		"--share=ipc",
+		"--device=dri",
+		"--talk-name=org.freedesktop.secrets",
```

Comment:
What exactly are this used for?
If I got it right, this option is only used when you need to talk to other processed using IPC, which I think is not needed in this case.

Reply:
We are getting org.freedesktop.Dbus error

Reply:
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

Reply:
Although I use those last 2 commands I'm getting error